kin-sdk/kin-sdk-lib/src/test/java/kin/sdk/FakeKeyStore.java

```diff
@@ -24,8 +25,13 @@
     }
 
     @Override
-    public void deleteAccount(int index) {
-        accounts.remove(index);
+    public void deleteAccount(String publicAddress) {
+        for (int i = 0; i < accounts.size(); i++) {
+            if (accounts.get(i).getAccountId().equals(publicAddress)) {
+                accounts.remove(i);
```

Comment:
removing in the middle of the loop can cause errors

Reply:
As far as I know, it is correct if you continue with the loop but in this case immediately after I found the item I remove and stop the loop.
It is the same as to find the index exit the loop and then removing, no?
And it is in the tests only.

Reply:
all cool, missed the break, need new 👓

Reply:
Already update the code, take a look